mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -301,12 +301,12 @@ CLASS net/minecraft/server/MinecraftServer net/minecraft/server/MinecraftServer
 		ARG 1 serverPort
 	METHOD m_prhzfzen getDefaultGameMode ()Lnet/minecraft/unmapped/C_lghcpyvl;
 	METHOD m_pwdmbosg reloadResources (Ljava/util/Collection;)Ljava/util/concurrent/CompletableFuture;
-		COMMENT Reloads this server's datapacks.
+		COMMENT Reloads this server's dataPacks.
```

Comment:
```suggestion
		COMMENT Reloads this server's data-packs.
```

This is javadoc, not code...

Reply:
I just asked whether this should be modified as well, but thanks for the clear response!